vue-frontend/src/components/services/WhatWeOfferDesktop.vue

```diff
@@ -61,11 +61,12 @@
 </template>
```

Comment:
Also, let's check page performance and attach the page score in the MR description.